admin/src/tests/helpers/TestUtils.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * Flush all pending resolved promise handlers.
+ */
+export const flushPromises = () => new Promise(resolve => setImmediate(resolve));
```

Comment:
@fjcero did you notice I also added a helper and it's already merged to master. Can you rebase and make sure we only have one TestUtils?